fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/BoundRecordQuery.java

```diff
@@ -134,7 +134,7 @@ public int hashCode() {
 
     @SuppressWarnings("UnstableApiUsage")
     private int computeHashCode() {
-        final HashFunction hashFunction = Hashing.murmur3_32();
+        final HashFunction hashFunction = Hashing.murmur3_32_fixed();
```

Comment:
I suspect you know this better than me, but this hash function is slightly different from before (due to a bug fix), so if there are problems caused by this function being unstable across releases, it's possible we need to continue using the older (deprecated) function.